src/Entity/Activity.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ActivityRepository")
+ */
+class Activity
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=1000)
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $applicationDeadline;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $finalDeadline;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $status;
```

Comment:
What will be the values for this field? For status it's better to have constants as values:

```
public const STATUS_NEW = 0;
public const STATUS_FINISHED = 1;
public const STATUS_CLOSED = 2;

private $status = self::STATUS_NEW;
```

Reply:
What type should we use for fields with constants?